coral/src/app/features/approvals/topics/components/TopicApprovalsTable.tsx

```diff
@@ -28,21 +31,31 @@ type TopicApprovalsTableProp = {
       topicId: number | null;
     }>
   >;
+  setRejectModal: Dispatch<
+    SetStateAction<{
+      isOpen: boolean;
+      topicId: number | null;
+    }>
+  >;
+  approveRequest: UseMutateFunction<
+    GenericApiResponse[],
+    HTTPError,
+    { requestEntityType: "TOPIC"; reqIds: string[] }
+  >;
+  approveIsLoading: boolean;
 };
 function TopicApprovalsTable(props: TopicApprovalsTableProp) {
-  const { requests, setDetailsModal } = props;
+  const {
+    requests,
+    setDetailsModal,
+    setRejectModal,
+    approveRequest,
+    approveIsLoading,
+  } = props;
 
   const columns: Array<DataTableColumn<TopicRequestTableRow>> = [
     { type: "text", field: "topicname", headerName: "Topic" },
-    {
-      type: "status",
-      field: "environmentName",
-      headerName: "Environment",
-      status: ({ environmentName }) => ({
-        status: "neutral",
-        text: environmentName,
-      }),
-    },
+    { type: "text", field: "environmentName", headerName: "Environment" },
```

Comment:
🤔  The environment should be chips (in neutral, because we can't really assign color right now), why is this removed? Was there a design decision I missed?

Reply:
That is probably a bad rebase :o

Reply:
[6009269](https://github.com/aiven/klaw/pull/694/commits/6009269f818915f3415f14098c033a38b8fca147) Thank you for noticing

Reply:
PHEW I thought I have missed something! :D awesome, thanks for updating!